src/styles/blocks/card.scss

```diff
@@ -0,0 +1,81 @@
+.card {
+  display: flex;
+  max-width: $cardWidth;
+  flex-direction: column;
+  background: $cardBackgroundColor;
+  border: 1px solid $cardBorderColor;
+  border-radius: 5px;
+  padding: 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 16px 3px 0;
+  }
+
+  &__title {
+    margin: 40px auto 0;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: $lineHeightForCard;
+    color: $mainTextColor;
+  }
+
+  &__subtitle {
+    margin: 4px 0 0;
+    color: $secondaryTextColor;
+  }
+
+  &__raiting {
+    display: flex;
+    justify-content: space-between;
+    margin: 16px 0 0;
+  }
+
+  &__reviews {
+    color: $mainTextColor;
+  }
+
+  &__prices {
+    margin: 24px 0 0;
+    display: flex;
+    justify-content: space-between;
+  }
+
+  &__price {
+    font-size: 12px;
+    line-height: $lineHeightForCard;
+    color: $secondaryTextColor;
+  }
+
+  &__value {
+    font-weight: $fontWeightForCard;
+    font-size: 16px;
+    line-height: $lineHeightForCard;
+    color: $mainTextColor;
+  }
+
+  &__link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-transform: uppercase;
+    margin: 16px 0 0;
+    height: 40px;
+    background: $cardLinkColor;
+    border-radius: 5px;
+    color: $cardLinkTextColor;
+    font-weight: $fontWeightForCard;
+    font-size: 14px;
+    line-height: 16px;
+    text-decoration: none;
+    border: 1px solid transparent;
+    transition: border-color 1s, color 1s, background-color 1s;
+
+    &:hover {
+      border-color: $linkHoverBorderColor;
```

Comment:
why do we need change border color? we can define color in `card__link` and it's enough